client/src/components/DatasetInformation/mount.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Endpoint for mounting job metrics from non-Vue environment.
+ */
+import $ from "jquery";
```

Comment:
Could you do this without jquery please?

Reply:
It is temporary code, this mount.js file will be gone as soon as we convert show_params.mako to Vue. This is just a way to mount Vue inside .mako file.

Reply:
@dannon but yes... I can do this without jquery